src/ProjectTemplates/Web.ProjectTemplates/RazorClassLibrary-CSharp.csproj.in

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Razor">
 
   <PropertyGroup>
-    <TargetFramework Condition="'$(SupportPagesAndViews)' == 'True'">netcoreapp3.0</TargetFramework>
+    <TargetFramework Condition="'$(SupportPagesAndViews)' == 'True'">netcoreapp3.1</TargetFramework>
```

Comment:
Ah must have been a one off debugging run where I modified this file to confirm that it was the cause of template failures. And then when I did a search and replace, this wasn't picked up since it was no longer netcoreapp3.0. Good catch.